scripts/run-integration-tests.sh

```diff
@@ -4,12 +4,14 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+readonly jdbc_driver_version=7.1.10
```

Comment:
Hardcoding the number here is not ideal. Use `grep` + `sed` to extract it from more central file if possible.

Reply:
This is the only place where the version number is specified.
I could get the Exasol version from the GH Action yaml, but not all Exasol versions have their own driver.
Not sure how to handle this in another way.

Reply:
If this is the only place, then the solution is fine.